101-vm-sql-full-autobackup/README.md

```diff
@@ -0,0 +1,43 @@
+# This template will create a SQL Server 2014 SP1 Enterprise edition with Auto Backup feature enabled.
```

Comment:
Let's make the full templates a 201 level since this requires a bit more understanding